docs/tutorials/vscode_devcontainers_tutorial.md

```diff
@@ -0,0 +1,187 @@
+- Next you need to make some modifications to the `devcontainer.json` file that is provided to you by the extension in order to get it working with Podman
+	- if you are running a Linux machine, you need to have the extension mount your workspace with the proper SELinux context. This context is what access control is based off of. You additionally need to map your current uid and gid to the same values in the container by creating a user namespace:
+	``` json
+	// Add the following lines to your devcontainer.json file to mount the workspace with the proper SELinux context
+	"workspaceMount": "source=${localWorkspaceFolder},target=/workspace,type=bind,Z",
+	"workspaceFolder": "/workspace",
+
+	// Add the following lines to your devcontainer.json file to map your current uid and gid to the container
+	"runArgs": ["--userns=keep-id"],
+	"containerUser": "vscode",
+	```
+- After making those modifications, you can run your first development container! Using the Command Palette in Visual Studio Code, begin typing and select the option "Dev Containers: Rebuild and Reopen in Container"
+	- This will begin setting up your development container
+	- Once the extension is done setting up your container, pull up the terminal and you can interact with the container! Feel free to create files and make changes to them. 
+
+### Set up your blog source code
+
+Create a new directory on your Desktop and add a file inside named `index.md` with these contents:
+
+```md
+# My blog
+
+Here is my blog about horses.
+```
+
+### Add devcontainer
+
+Add a folder inside your project folder named `.devcontainer`. And inside add these files:
+
+`devcontainer.json`
+
+```json
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.241.1/containers/jekyll
+{
+	"name": "Jekyll",
+	"build": {
+		"dockerfile": "Dockerfile",
+		"args": {
+			// Update 'VARIANT' to pick a Debian OS version: bullseye, buster
+			// Use bullseye when on local arm64/Apple Silicon.
+			"VARIANT": "bullseye",
+			// Enable Node.js: pick the latest LTS version
+			"NODE_VERSION": "lts/*"
+		}	
+	},
+
+	// Use 'forwardPorts' to make a list of ports inside the container available locally.
+	"forwardPorts": [
+		// Jekyll server
+		4000,
+		// Live reload server
+		35729
+	],
+
+	// Use 'postCreateCommand' to run commands after the container is created.
+	"postCreateCommand": "sh .devcontainer/post-create.sh",
+
+	// Comment out to connect as root instead. More info: https://aka.ms/vscode-remote/containers/non-root.
+	"remoteUser": "vscode",
+
+	"workspaceMount": "source=${localWorkspaceFolder},target=/workspace,type=bind,Z",
+	"workspaceFolder": "/workspace",
+
+	"runArgs": ["--userns=keep-id"],
+	"containerUser": "vscode"
+}
+```
+
+`Dockerfile`
```

Comment:
I'm assuming it has to be a Dockerfile vs a Containerfile

Reply:
confirmed that Containerfile does work